independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/update/QuarkusUpdateRecipe.java

```diff
@@ -0,0 +1,76 @@
+package io.quarkus.devtools.project.update;
+
+
+import io.quarkus.devtools.project.BuildTool;
+import io.quarkus.devtools.project.update.operations.UpdatePropertyOperation;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class QuarkusUpdateRecipe {
+
+    public static final Map<String, Object> QUARKUS_RECIPE = Map.of(
+        "type", "specs.openrewrite.org/v1beta/recipe",
+        "name", "io.quarkus.openrewrite.Quarkus",
+        "displayName", "Migrate quarkus project to a new version",
+        "description", "Update Quarkus version and refactor imports and resources if needed.",
+        "tags", List.of("quarkus")
+    );
+
+    public static final String RECIPE_LIST_KEY = "recipeList";
+    private BuildTool buildTool = BuildTool.MAVEN;
+    private final List<Operation> operations = new ArrayList<>();
+    private final List<Map<String, Object>> otherRecipes = new ArrayList<>();
+
+    public QuarkusUpdateRecipe() {
+    }
+
+    public QuarkusUpdateRecipe setBuildTool(BuildTool buildTool) {
+        this.buildTool = buildTool;
+        return this;
+    }
+
+    public QuarkusUpdateRecipe addOperation(Operation operation) {
+        this.operations.add(operation);
+        return this;
+    }
+
+    @SuppressWarnings("unchecked")
+    public QuarkusUpdateRecipe addOtherRecipes(List<Object> recipe) {
+        for (Object r : recipe) {
+            if (r instanceof Map) {
+                addOtherRecipe((Map<String, Object>) r);
+            }
+        }
+        return this;
+    }
+
+    public QuarkusUpdateRecipe addOtherRecipe(Map<String, Object> recipe) {
+        Objects.requireNonNull(recipe, "recipe is required");
+        if(!recipe.containsKey("name") || !(recipe.get("name") instanceof String)) {
+            throw new IllegalArgumentException("Recipe name is required");
+        }
+        this.otherRecipes.add(recipe);
+        return this;
+    }
+
+    public BuildTool getBuildTool() {
+        return buildTool;
+    }
+
+    public List<Operation> getOperations() {
+        return operations;
+    }
+
+    public List<Map<String, Object>> getOtherRecipes() {
```

Comment:
I don't see a `getRecipes`, thus what is `getOtherRecipes` ?

Reply:
Fixed